CMakeLists.txt

```diff
@@ -401,8 +403,9 @@ set(plugins
   gazebo_catapult_plugin
   gazebo_usv_dynamics_plugin
   gazebo_parachute_plugin
-  gazebo_airship_dynamics_plugin
   gazebo_drop_plugin
+  gazebo_windsensor_plugin
```

Comment:
This is not really a wind sensor, right? for example, when the vehicle is moving it is measuring the relative airspeed and direction not the wind?

Reply:
The FT Technologies sensor, measures the airflow magnitude in the X-Y plane of the sensor and then provides the direction of this flow. Thus if the sensor is mounted on a vehicle it will measure the combined airflow produced by the vehicle's groundspeed + the world's wind flow. 

The correction for the vehicle's movement will thus need to be made after sampling the raw sensor data. 

Happy to proceed with any new naming suggestions here and in mavlink.